code/game/machinery/engi_points_vendor.dm

```diff
@@ -452,6 +452,7 @@
 	var/stop = 0
 	var/beat = 7
 	var/duration = 600
+	var/list/available = list()
 	var/static/list/choices = list("Engineering's Basic Beat", "Engineering's Domination Dance", "Engineering's Superiority Shimmy", "Engineering's Ultimate High-Energy Hustle")

```

Comment:
Thanks for showing how to make those lists, but my other issue is how to use them... how do I pull the 3rd item from the 2nd list, etc.

I had to deal with lists inside lists for alarms and even after like 6-7 tries I was basically hooked on Coderbus IRC for life support because I kept screwing it up.